lib/consts/Enums.ts

```diff
@@ -8,6 +8,23 @@ export enum ErrorCodePrefix {
   Swap = 6,
 }
 
+export enum Symbols {
```

Comment:
The name of enums should always be singular 

```suggestion
export enum Symbol {
```